.github/workflows/node.js.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [16.x]
```

Comment:
Are we updating the node version in package.json also?

Reply:
yes